tests/test_SklearnPipelinePipeline.py

```diff
@@ -0,0 +1,93 @@
+        model.fit(data, y)
+        model_onnx = convert_sklearn(model, 'pipelinepipeline', [('input', FloatTensorType([1, 2]))])
+        self.assertTrue(model_onnx is not None)
+        dump_data_and_model(data, model, model_onnx, basename="SklearnPipelinePcaPipelineMinMax")
+        
+    def test_pipeline_pca_pipeline_none_lin(self):
+        
+        model = Pipeline(memory=None,
+                    steps=[('PCA', PCA(copy=True, iterated_power='auto',
+                            n_components=0.15842105263157896, random_state=None,
+                            svd_solver='auto', tol=0.0, whiten=False)), 
+                           ('Pipeline', Pipeline(memory=None,
+                                steps=[('MinMax scaler', MinMaxScaler(copy=True, feature_range=(0, 3.7209871159509307))),
+                                       ('logreg', LogisticRegression())]))])
+
+        data = numpy.array([[0, 0], [0, 0], [1, 1], [1, 1]], dtype=numpy.float32)
+        y = [0, 0, 1, 1]
+        model.fit(data, y)
+        model_onnx = convert_sklearn(model, 'pipelinepipeline', [('input', FloatTensorType([1, 2]))])
+        self.assertTrue(model_onnx is not None)
+        dump_data_and_model(data, model, model_onnx, basename="SklearnPipelinePcaPipelineMinMaxLogReg")
+        
+    def test_pipeline_pca_pipeline(self):
+        
+        model = Pipeline(memory=None,
+                    steps=[('PCA', PCA(copy=True, iterated_power='auto',
+                            n_components=2, random_state=None,
+                            svd_solver='auto', tol=0.0, whiten=False)), 
+                           ('Pipeline', Pipeline(memory=None,
+                                steps=[('MinMax scaler', MinMaxScaler(copy=True, feature_range=(0, 3.7209871159509307))),
+                                       ('MultinomialNB', MultinomialNB(alpha=0.7368421052631579, class_prior=None, fit_prior=True))]))])
+
+        data = numpy.array([[0, 0, 0], [0, 0, 0.1], [1, 1, 1.1], [1, 1.1, 1]], dtype=numpy.float32)
+        y = [0, 0, 1, 1]
+        model.fit(data, y)
+        model_onnx = convert_sklearn(model, 'pipelinepipeline', [('input', FloatTensorType([1, 2]))])
+        self.assertTrue(model_onnx is not None)
+        dump_data_and_model(data, model, model_onnx, basename="SklearnPipelinePcaPipelineMinMaxNB2")
+        
+    def test_pipeline_pca_pipeline_none(self):
+        
+        model = Pipeline(memory=None,
+                    steps=[('PCA', PCA(copy=True, iterated_power='auto',
+                            n_components=0.15842105263157896, random_state=None,
+                            svd_solver='auto', tol=0.0, whiten=False)), 
+                           ('Pipeline', Pipeline(memory=None,
+                                steps=[('MinMax scaler', MinMaxScaler(copy=True, feature_range=(0, 3.7209871159509307))),
+                                       ('MultinomialNB', MultinomialNB(alpha=0.7368421052631579, class_prior=None, fit_prior=True))]))])
```

Comment:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
We should Apply pep8 on all files in one row. On my projects, I usually add a unit test which checks the syntax.

Reply:
Yes, we should. Let's start with abiding by pep8 conventions in whatever we check-in from here on and create a separate task to update the previously committed code that violates pep8.

Reply:
scikit-learn enforce pep8 on every diff.